### ISSUE TITLE
Missing Debian packages in releases

### ISSUE THREAD

**Issue by nop33** [drakkan/sftpgo#182]
The documentation mentions:

> Binary releases for Linux, macOS, and Windows are available. Please visit the releases page.
> ...
> Deb and RPM packages are built after each commit and for each release.

but I cannot find any `.deb` files in the [releases page](https://github.com/drakkan/sftpgo/releases). Only `tar.xz` files.

**Comment by drakkan** [comment]
Hi,

a debian package will be automatically built when I'll push the next tag. 
It should happen before the end of this week. 
Meatime you can download deb/rpm from the "Actions" page, just select a commit and download the binaries for it.

 Thank you

**Comment by nop33** [comment]
Thank you for your quick reply!

I am getting the URL of the `.deb` package with right-click on the _sftpgo-v1.0.1-dev.77-x86_64-deb_ link found in the [Actions](https://github.com/drakkan/sftpgo/actions/runs/289114329) page and trying to download it on my server with `wget` but all I get is a file called `20083370`...
``` console
$ wget https://github.com/drakkan/sftpgo/suites/1289357855/artifacts/20083370
```
Do you know how else I can download it using the CLI? I know this question is not related to the project but I thought it's worth asking anyway...

**Comment by drakkan** [comment]
you need to be logged in with a GitHub account to download the build artififacts (this is how GitHub work), so probably you have to download the deb to your pc and then transfer it to your server

**Comment by nop33** [comment]
I see. Thanks again!

**Comment by drakkan** [comment]
deb package is now available for v1.1.0. There is a [PPA](https://launchpad.net/~sftpgo/+archive/ubuntu/sftpgo) too
